source/includes/_endpoint-user.md

```diff
@@ -76,4 +106,64 @@ The new owner must also be part of the same account and should have
 }
 ````
 
-The new owner provided can be a user URL or a user email.
\ No newline at end of file
+The new owner provided can be a user URL or a user email.
+##### User Datasets
+
+`/account/users/{id}/datasets/`
+
+This URL is only accessible and available to account admins. 
+
+This Shoji catalog lists all the datasets that are owned by this user.
+
+
+##### User Visible datasets
+
+`/users/{id}/visible_datasets/`
```

Comment:
We do have a bit of a mix up here. Following the shoji path `user.datasets` actually points to `/account/users/id/datasets/` which shows the datasets owned by that user. 

We have 3 lists to show (from cheapest to most expensive to calculate):
 * Datasets owned by a user, currently under /account/users/id/datasets/
 * Authenticated user's datasets catalog that contains owned and direct shares under /dataset/
 * All datasets visible by a user that on top of the dataset datalog includes team and project access datasets all in one list, currently under /visible_datasets/ and only account admins can peek at this

They all serve different purposes. 

For simplicity in the API I could move /account/users/id/datasets/ to live straight under /users/id/datasets/ and then add the /visible/ sub catalog under that. We'd still have the 3 catalogs but the paths may be less convoluted. Right now there is little value on keeping /account/users/id/datasets/ under /accounts/ and requires a hacky transcient entity on /account/users/id/